src/main/java/io/nozemi/runescape/content/magic/NormalTeleports.java

```diff
@@ -33,4 +33,94 @@ public void homeTeleport(Player player, int option, int item) {
             player.teleport(new Tile(2606, 3102));
         }
     }
+    @Button(parentId = 218, childId = 16)
+    public void varrockTeleport(Player player, int option, int item) {
+        Optional<MyTeleports> teleport = TeleportEffectChainHandler.getTeleport("VARROCK");
```

Comment:
Use LUMBRIDGE for all teleports for now, because there are no others configured. We'll likely configure the default teleport effect sequence later.

Reply:
Rewrite it in to 5 animations, Home, Modern, Ancient, Lunar, Arcuees. Must still write the 4 spellbooks, only home is writen.